src/acceptance-tests-brain/test-scripts/011_nfspersi_test.rb

```diff
@@ -10,6 +10,11 @@
 
 use_global_timeout
 
+puts "#{c_bold}Test disabled.#{c_reset}"
+puts "#{c_bold}Depends on CAP-1293#{c_reset} to enable NFS persi support."
```

Comment:
Do we have a GitHub reference here? (Jira is not a good place to keep things with a public impact.)

Reply:
Not aware of any (I looked into the CAP, just rechecked, no such link)

Reply:
Found ticket made by @gak: cloudfoundry-incubator/kubecf#382
Added to notice, and CAP ticket.

Reply:
Addressed with commit f2be8c95181e3a6e5e099501d59df6ae6a4d7c48.